CHANGES.rst

```diff
@@ -10,8 +10,37 @@ New Features in 24.0
 - A new log level called ``CONSOLE`` has been added between the default
   ``INFO`` and ``DEBUG`` levels. This level will show all reads and writes made
   to the serial console during testing.
-- The `QEMUDriver` now has an additional ``disk_opts`` property which can be
+- The docker support was extended to support buildx, allowing the build of arm64
+  container images.
+- The tool lookup function has been extended to return the original name in case
+  the path can't be found. This makes specification of the qemu binary to use
+  easier.
+- The bindings baseclass has been extended, allowing the user to retrieve all
+  resources used by a driver.
+- Support for STLink V2 was added.
+- UBootStrategy was extended with a force() function.
+- labgrid was switched from pysnmp to pysnmp-lexstudio.
+- Support for Segger J-Link was added.
+- Place tags are now exposed by the RemotePlace.
+- The sync-places contrib script has gained support for named matches.
+- Remote support for YKush Devices was added.
+- Support for sigrok DMMs was added.
+- Support for Digital Outputs switched via HTTP was added.
+- The ``QEMUDriver`` has a new get_qemu_base_args() function which can be used to
+  extract the arguments passed to qemu.
+- The ``SSHDriver`` has gained support to forward unix sockets.
+- The exporter has gained an ``--fqdn`` argument to set the hostname to the
+  fully qualified domain name instead of the hostname.
+- The ``QEMUDriver`` now has an additional ``disk_opts`` property which can be
   used to pass additional options for the disk directly to QEMU
+- All drivers now inherit a logger from the ``Driver`` baseclass and many
+  drivers were changed to use this logger.
+- The new ``poe_mib`` backend allows switching of power over ethernet capable
+  ports on switches that use the corresponding SNMP MIB.
+- The ``RawNetworkInterfaceDriver`` allows the replay and recording of network
+  packets on ethernet interfaces.
+- The i.MX93 usb loader ID has been added.
```

Comment:
```suggestion
- The i.MX93 usb loader USB ID has been added.
```

Reply:
Maybe mention the corresponding resource?